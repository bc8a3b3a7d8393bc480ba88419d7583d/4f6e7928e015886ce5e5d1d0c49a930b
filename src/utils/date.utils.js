import moment from 'moment';

const supportedDateFormats = {
  ddddDDMMMYYYY: 'dddd, DD MMM YYYY',
};

const format = (date, dateFormat) => moment(date).format(dateFormat);

export default {
  format,
  supportedDateFormats,
};
