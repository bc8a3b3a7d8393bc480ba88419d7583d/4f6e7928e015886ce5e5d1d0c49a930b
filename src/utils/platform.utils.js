import { Platform } from 'react-native';
import DeviceInfo from 'react-native-device-info';
import { equals } from 'ramda';

const isDEV = () => equals(process.env.NODE_ENV, 'development');
const isIOS = () => equals(Platform.OS, 'ios');
const isAndroid = () => equals(Platform.OS, 'android');
const iPhoneX = 'iPhone X';
const isIPhoneX = () => isIOS() && equals(DeviceInfo.getModel(), iPhoneX);

export { isDEV, isIOS, isAndroid, isIPhoneX, iPhoneX };
