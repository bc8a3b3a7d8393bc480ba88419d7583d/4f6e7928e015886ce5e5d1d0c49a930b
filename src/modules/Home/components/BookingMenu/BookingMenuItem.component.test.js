import { testSnapshots } from '../../../../utils/test.utils';

import Component from './BookingMenuItem.component';

describe('Modules/Home - BookingMenuItem', () => {
  testSnapshots(Component, [
    {
      description: 'basic render',
      props: {
        onPress: jest.fn(),
        title: 'Flights',
        image: 'flight',
      },
    },
  ]);
});
