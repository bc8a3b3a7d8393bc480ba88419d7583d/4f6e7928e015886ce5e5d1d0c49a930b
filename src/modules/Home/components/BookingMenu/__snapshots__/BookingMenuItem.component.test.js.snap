// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modules/Home - BookingMenuItem snapshots basic render 1`] = `
<TouchableWithoutFeedback
  onPress={[MockFunction]}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderRadius": 10,
        "elevation": 4,
        "flex": 1,
        "justifyContent": "center",
        "margin": 3,
        "maxHeight": 70,
        "minWidth": 90,
        "padding": 10,
        "shadowColor": "#8f8f8f",
        "shadowOffset": Object {
          "height": 1,
          "width": 1,
        },
        "shadowOpacity": 0.3,
      }
    }
  >
    <Image
      resizeMode="cover"
      source={
        Object {
          "testUri": "../../../src/assets/images/home/flight.png",
        }
      }
      style={
        Object {
          "backgroundColor": "white",
          "borderRadius": 17.5,
          "height": 35,
          "marginBottom": 5,
          "width": 35,
        }
      }
    />
    <Text
      style={
        Object {
          "color": "black",
          "fontFamily": "Oswald-Regular",
          "fontSize": 10,
        }
      }
    >
      Flights
    </Text>
  </View>
</TouchableWithoutFeedback>
`;
